test/data_test.go

```diff
@@ -174,17 +179,18 @@ port: 80
 headers:
 User-Agent: "ModSecurity CRS 3 Tests"
 Host: "localhost"
-Content-Type: "application/x-www-form-urlencoded"
+Content-Type: header_values.ApplicationXWwwFormUrlencoded
```

Comment:
++

If we use it so many times, do we need to repeat it this much? 🤔

Reply:
The strings have small differences, so it's not so easy to extract a template without parameterizing everything.